README.md

```diff
@@ -143,18 +158,21 @@ with Installer(
     installation.create_users(user)
 ```
 
-This installer will perform the following actions:
-
-* Prompt the user to configure the disk partitioning
-* Prompt the user to setup disk encryption
-* Create a file handler instance for the configured disk and the optional disk encryption
-* Perform the disk operations (WARNING: this will potentially format the disks and erase all data)
-* Install a basic instance of Arch Linux *(base base-devel linux linux-firmware btrfs-progs efibootmgr)*
-* Install and configures a bootloader to partition 0 on UEFI. On BIOS, it sets the root to partition 0.
-* Install additional packages *(nano, wget, git)*
-* Create a new user
-
-> **To create your own ISO with this script in it:** Follow [ArchISO](https://wiki.archlinux.org/index.php/archiso)'s guide on creating your own ISO.
+### This installer will perform the following actions
+
+- Prompt the user to
```

Comment:
I personally don't think this is more readable and it's just creating sub-categories for no reason